transmit-lib/src/cmr/transmit/http_helper.clj

```diff
@@ -102,24 +104,30 @@
   * :raw? - indicates whether the raw HTTP response (as returned by http-response->raw-response ) is
   desired. Defaults to false.
   * :use-system-token? - indicates if the ECHO system token should be put in the header
-  * :http-options - a map of additional HTTP options to send to the clj-http.client/request function.
+  * :http-options - map of additional HTTP options to send to the clj-http.client/request function.
   * :response-handler - a function to handle the response. Defaults to default-response-handler"
-  [context app-name {:keys [url-fn method http-options response-handler use-system-token?] :as request}]
+  [context app-name {:keys [url-fn method http-options response-handler use-system-token?]
+                     :as request}]
   (let [conn (config/context->app-connection context app-name)
         response-handler (or response-handler default-response-handler)
         connection-params (config/conn-params conn)
-        ;; Validate that a connection manager is always present. This can cause poor performance if not.
+        ;; Validate that connection manager is always present. This causes poor performance if not.
         _ (when-not (:connection-manager connection-params)
-            (errors/internal-error! (format "No connection manager created for [%s] in current application" app-name)))
+            (errors/internal-error!
```

Comment:
this section, the request function, is pretty central, take a good look at how the http-options are built and point out anything you all think is not going to work.